engine/runtime-integration-tests/src/test/java/org/enso/compiler/test/StaticAnalysisTest.java

```diff
@@ -0,0 +1,138 @@
+package org.enso.compiler.test;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.OutputStream;
+import java.util.List;
+import java.util.logging.Level;
+import org.enso.common.LanguageInfo;
+import org.enso.common.MethodNames;
+import org.enso.common.RuntimeOptions;
+import org.enso.compiler.core.IR;
+import org.enso.compiler.core.ir.Diagnostic;
+import org.enso.compiler.core.ir.DiagnosticStorage;
+import org.enso.compiler.core.ir.Expression;
+import org.enso.compiler.core.ir.Module;
+import org.enso.compiler.core.ir.module.scope.definition.Method;
+import org.enso.editions.LibraryName;
+import org.enso.interpreter.runtime.EnsoContext;
+import org.enso.interpreter.test.InterpreterContext;
+import org.enso.pkg.QualifiedName;
+import org.graalvm.polyglot.Source;
+import scala.jdk.javaapi.CollectionConverters;
+
+public abstract class StaticAnalysisTest {
```

Comment:
Methods in this class seems useful even outside of static analysis. Could you try either merging them to [ContextUtils](https://github.com/enso-org/enso/blob/fb821303ab2818c4ce2d8482612584497f2e2014/lib/java/test-utils/src/main/java/org/enso/test/utils/ContextUtils.java) or just to the same package and refactoring the methods here as static?

Reply:
I will see what I can do.

This PR was in progress for such a long time (because I was rarely finding time to work on it), that I think `StaticAnalysisTest` was created before `ContextUtils`. I guess if I were creating it now I'd indeed put them somewhere around there.